debian/control

```diff
@@ -4,19 +4,23 @@ Section: misc
 Priority: optional
 Standards-Version: 3.9.8
 Build-Depends: cmake,
-               debhelper (>= 9),
-               libbrotli-dev,
-               libgif-dev,
-               libgmock-dev,
-               libgoogle-perftools-dev,
-               libgtest-dev,
-               libhwy-dev,
-               libjpeg-dev,
-               libopenexr-dev,
-               libpng-dev,
-               libwebp-dev,
-               pkg-config,
-Homepage: https://gitlab.com/wg1/jpeg-xl
+ debhelper (>= 9),
```

Comment:
lets keep the list sorted. Was there a problem with the indentation?

Reply:
@deymo

* libhwy-dev – Everything I try to build against libjxl-dev fails when libhwy-dev is not also installed.  Nothing else currently  uses libhwy, except libjxl.  Looks like you found the specific files involved.
* doxygen, xmlto  – When trying to build the man pages on launchpad, I got various errors because of missing files that are provided these packages.  I think they were related to `lintian` checking `xml` output.  I would have to look for the logs if you want more details.
* xdg-utils – This is needed to provide `xdg-mime` referenced in `plugins/gdk-pixbuf/CMakeLists.txt`
* spacing – It was just annoying to have to type so many spaces every time I added a package to the depends list.  Most other packages seem to use just one space.
* alphabetization – I just added the packages to the end of the list in the order that I encountered and fixed build errors.  I will revise the commit to alphabetize.